README.md

```diff
@@ -1,30 +1,38 @@
 [![Build Status](https://travis-ci.org/mozilla/donate.mozilla.org.svg?branch=master)](https://travis-ci.org/mozilla/donate.mozilla.org)
 
 # donate.mozilla.org
-Mozilla donation forms
+Mozilla donation forms:
 
 * Staging: https://donate.mofostaging.net/
 * Production: https://donate.mozilla.org/
 
-## Requirements
+---
+
+## Pre-requisites
```

Comment:
I think there’s no need for a hyphen here, but will clearly not block on that for a readme file :)